pages/linux/makepkg.md

```diff
@@ -1,22 +1,22 @@
 # makepkg
 
-> Creates a package installable with the `pacman` package manager.
-> Runs the commands from a `PKGBUILD` file to build the package.
+> Creates a package which can be used with `pacman`.
+> Uses the `PKGBUILD` file in the current working directory by default.
```

Comment:
I think this sounds better
```suggestion
> Create a package which can be used with `pacman`.
> Use the `PKGBUILD` file in the current working directory by default.
```

Reply:
I agree with the first line however the second line change seems out of place.

Reply:
Yep, reading it on a new day, it does seem odd :(
We can maybe omit the `by default`, given we start with `Use`.

Reply:
I also agree with @CleanMachine1.

Reply:
Maybe this sounds better ? (thoughts)
```suggestion
> Creates a package which can be used with `pacman`.
> Uses the `PKGBUILD` file in the current working directory to build.
```

Reply:
I think axel's edited suggestion is sufficient